aws_jobs/cyhy-data-extract.py

```diff
@@ -136,30 +135,15 @@ def query_data(collection, query, tbz_file, tbz_filename, end_of_data_collection
     with open(json_filename, "w") as collection_file:
         collection_file.write("[")
 
-        last_id = None
-        while True:
-            if last_id is None:
-                result = collection.find(
-                    query, {"key": False}, sort=[("_id", pymongo.ASCENDING)]
-                ).limit(PAGE_SIZE)
-            else:
-                query["_id"] = {"$gt": last_id}
-                result = collection.find(
-                    query, {"key": False}, sort=[("_id", pymongo.ASCENDING)]
-                ).limit(PAGE_SIZE)
-
-            for doc in result:
-                collection_file.write(to_json([doc])[1:-2])
-                collection_file.write(",")
-
-            last_id = doc["_id"]
-
-            if result.retrieved == 0:
-                # If we output documents then we have a trailing comma, so we need to
-                # roll back the file location by one byte to overwrite as we finish
-                if last_id is not None:
-                    collection_file.seek(-1, os.SEEK_END)
-                break
+        result = collection.find(query, {"key": False})
+        for doc in result:
+            collection_file.write(to_json([doc])[1:-2])
+            collection_file.write(",")
+
+        if result.retrieved == 0:
```

Comment:
Is there a disconnect between `if result.retrieved == 0` and the comment below?  The comment begins with "If we output documents..." but doesn't `result.retrieved == 0` imply that we _did not_ output any documents?

It seems that maybe it should be `if result.retrieved != 0`, since that would be the case where we have a trailing comma.

Reply:
You are correct. I overlooked it while removing the pagination.

Reply:
Corrected by @mcdonnnj in 9b52d80.